torch/optim/adafactor.py

```diff
@@ -248,9 +256,6 @@ def _single_tensor_adafactor(
             col_mean = torch.norm(grad, dim=-2).square_().div_(grad.size(-2))
             col_var.lerp_(col_mean, 1 - beta2_t)
             var_estimate = row_var.unsqueeze(-1) @ col_var.unsqueeze(-2)
-            # Include in docs later: we move epsilon1 to these 2 lines
-            # as an attempt to follow the intention of the paper authors
-            # in using eps1 for stability.
             var_est_denom = (
                 row_var.mean(dim=-1).clamp_(min=eps1).unsqueeze(-1).unsqueeze(-1)
```

Comment:
adding `keepdim=True` can save one of `unsqueeze` calls probably

Reply:
Yea, keepdim is a good call for everywhere